docs/operator-manual/notifications/troubleshooting.md

```diff
@@ -16,7 +16,7 @@ Additionally, you can specify `:empty` to use empty secret with no notification
 
     ```bash
     argocd admin notifications trigger get \
-      --config-map ./argocd admin notifications-cm.yaml --secret :empty
+      --config-map notifications-cm.yaml --secret :empty
```

Comment:
Looking at the docker example in the same file this seems to be the likely command

```suggestion
      --config-map ./argocd-notifications-cm.yaml --secret :empty
```

Reply:
Oh thanks!